### DISCUSSION TITLE
Onramp Flow using Web3Auth

### DISCUSSION THREAD

**Discussion by JSeam2** [Web3Auth/Web3Auth#768] [Feature Requests]
Is there an onramp flow using web3auth? Some users may not have crypto already available but will want to buy crypto assets to interact with applications. How can we go about this?

**Comment by chaitanyapotti** [comment]
Yes.
Web3Auth supports topup with credit card via torus wallet plugin.
Docs on how to add plugin: https://web3auth.io/docs/sdk/web/plugins/torus-wallet
Docs on how to topup from plugin: https://docs.tor.us/wallet/api-reference/topup
